api/v1alpha1/node_pools.go

```diff
@@ -0,0 +1,89 @@
+/*
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in
+compliance with the License. You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is
+distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+implied. See the License for the specific language governing permissions and limitations under the
+License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type LocationSpec struct {
+	Location string `json:"location,omitempty"`
+	Site     string `json:"site"`
```

Comment:
Looking at the "O-RAN O2ims Interface Specification 5.0" I only see mention of the location (no mention of site). Lets just stick with "location" for now unless someone else has seen "site" in the O-RAN documents somewhere. I will update this in the design as well.

Reply:
Per Dell, the compute resource pool(s) are mapped to a site (while a site is created on a location). See reference: https://docs.google.com/document/d/19lPxbaQkb-V8JNL3JJXCMEqivNqGmucOF48IAu7FOCk/edit